Kickstarter-iOS/Views/Controllers/ManagePledgeViewControllerTests.swift

```diff
@@ -44,11 +44,17 @@ final class ManagePledgeViewControllerTests: TestCase {
       )
       |> \.backing.sequence .~ 10
       |> \.backing.location .~ ManagePledgeViewBackingEnvelope.Backing.Location(name: "United States")
+
+    envelope = envelope
```

Comment:
Had to split these out in a few places, was getting to complex for the compiler to parse 😓